### ISSUE TITLE
Source Model loader for OpenQuake DB

### ISSUE THREAD

**Issue by larsbutler** [gem/oq-engine#164]
Spec for: https://www.pivotaltracker.com/story/show/12546279
# Abstract

We need a way to load source model data from NRML files into the OpenQuake database. The code written for this story will use the SourceModelReader java class (called in python code using jpype). We should also be able to make use of sqlalchemy and geoalchemy to handle database serialization.
# What needs to be done?
- read source model data (using SourceModelReader) as jpype java objects
- there are 4 types of source models we need to read:
  - simple faults
  - complex faults
  - area sources
  - point sources
- massage the model data as necessary
- serialize to the database
# Solution outline
- Create a SourceModelLoader class (in python)
- This will call the SourceModelReader class using jpype
- We should be able to use sqlalchemy/geoalchemy to handle most of the db serialization work
  - Basically, the source model data just needs to be converted to a specially formatted dict (which can be then be written to the db by sqlalchemy
- The most difficult problem I anticipate is getting the sqlalchemy serialization utilities to work properly (and hopefully, transparently) with the geometry type data that we need to insert
# Test data needed
- java_tests/data/source_model.xml
- I'll also use the various smoketest source model files as examples
